Documentation/faq.md

```diff
@@ -2,6 +2,14 @@
 
 ### etcd, general
 
+#### When to use etcd
```

Comment:
this should be in the why etcd section that @fanminshi is working on now. do not put this in faq